README.md

```diff
@@ -45,6 +45,15 @@ npm run serve
 
 You can now access Bee Dashboard on [http://localhost:8080/](http://localhost:8080/)
 
+## Docker
+
+From inside project directory:
+
+```sh
+docker build . -t bee-dashboard
+docker run -p 8080:8080 bee-dashboard
+```
```

Comment:
```suggestion
To build Docker image, execute the following from inside project directory:

```sh
docker build . -t bee-dashboard
docker run --rm -p 127.0.0.1:8080:8080 bee-dashboard
```
```

Reply:
Missing the code block closing marker ```

Reply:
It's not, but github thinks it is the opening of next codeblock :/